scipy/spatial/_spherical_voronoi.py

```diff
@@ -140,10 +167,12 @@ class SphericalVoronoi:
     the coordinate system as the fourth vertex of each simplex of the Convex
     Hull. The circumcenters of all tetrahedra in the system are calculated and
     projected to the surface of the sphere, producing the Voronoi vertices.
-    The Delaunay tetrahedralization neighbour information is then used to
-    order the Voronoi region vertices around each generator. The latter
-    approach is substantially less sensitive to floating point issues than
-    angle-based methods of Voronoi region vertex sorting.
+    Alternatively, the centroids of each simplex of the Convex Hull are used
+    to establish the Voronoi vertices. The Delaunay tetrahedralization
```

Comment:
The centroids of the simplices of the Convex Hull are not guaranteed to represent the Voronoi vertices, even after projection to the surface of the sphere.

I suspect based on your mailing list message that you're actually looking for an approximation that works well at estimating the vertices sometimes (but certainly not in all cases).

If I'm understanding you correctly (?) this would probably need to be clarified as a performance-related *approximation* of the the true Voronoi vertices in the docstring & probably accompanied by some benchmarks (we tend to use `asv` to monitor regressions) to demonstrate the speed improvement vs. the true Spherical Voronoi diagram.